File: client/WorkoutLoggerJS/src/com/jessewarden/workoutlogger/vo/SetVO.js
define([], function()
{

	function SetVO(name, goalReps, goalWeight, reps, weight, goodForm)
	{
		this.name       = name;
		this.goalReps   = goalReps;
		this.goalWeight = goalWeight;
		this.reps       = reps;
		this.weight     = weight;
		this.goodForm   = goodForm;
	}

	return SetVO;

});

File: client/WorkoutLoggerDartAngular2/web/com/jessewarden/workoutlogger/vo/SetVO.dart
part of workoutloggerlib;

class SetVO
{
  String name;
  bool goodForm;
  int reps;
  double weight;
  int goalReps;
  double goalWeight;
  
	SetVO({String name, bool goodForm, int reps, double weight, int goalReps, double goalWeight})
	{
		this.name = name;
		this.goodForm = goodForm;
		this.reps = reps;
		this.weight = weight;
		this.goalReps = goalReps;
		this.goalWeight = goalWeight;
	}
  
}